app/steps/ui/deceased/name/index.js

```diff
@@ -1,16 +1,31 @@
 'use strict';
 
 const ValidationStep = require('app/core/steps/ValidationStep');
+const FeatureToggle = require('app/utils/FeatureToggle');
 
 class DeceasedName extends ValidationStep {
 
     static getUrl() {
         return '/deceased-name';
     }
 
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        ctx.isToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'screening_questions');
+        return [ctx, errors];
+    }
+
+    nextStepOptions() {
+        return {
+            options: [
+                {key: 'isToggleEnabled', value: true, choice: 'toggleOn'}
+            ]
+        };
+    }
+
```

Comment:
It would be less verbose to use a variable here rather than repeating the `return` statement, for example:

```
nextStepOptions(ctx) {
    const optionChoice = ctx.isToggleEnabled ? 'noCodicilsToggleOn' : 'noCodicils';
    return {
        options: [
            {key: 'codicils', value: json.optionNo, choice: optionChoice}
        ]
    };
}
```

It could be that you did it this way because you thought it would be easier and less error prone to clean up the feature toggle code?